### ISSUE TITLE
Doesn't work on big endian hosts

### ISSUE THREAD

**Issue by ghost** [foliojs/linebreak#8]
Hi @devongovett 
Linebreak works on big endian machines only if classes.trie is generated on big endian machine too. Otherwise there is an error
```
reak/src/linebreaker.js:122

switch (pairTable[this.curClass][this.nextClass]) {

^

TypeError: Cannot read property 'undefined' of undefined
```
Seen by @lcurcio at "IBM I" POWER system and by me on Debian MIPS

**Comment by liborm85** [comment]
Will be fixed in https://github.com/foliojs/unicode-trie/pull/5.

**Comment by devongovett** [comment]
Merged foliojs/unicode-trie#5. Closing.
